msal/application.py

```diff
@@ -701,6 +705,12 @@ def _validate_ssh_cert_input_data(self, data):
                     "you must include a string parameter named 'key_id' "
                     "which identifies the key in the 'req_cnf' argument.")
 
+    def _get_client(self):
```

Comment:
Here comes a totally subjective suggestion. I think it would be better to move this method higher in this source file, to right below the pre-existing `_build_client()`. This way, the 2 correlated methods stay together, thus easier to read and understand for future maintainers. They will thank us. :-)

Reply:
I think thats a good suggestion, will move it up in the next commits